### PR TITLE
Converts blob overmind shield to update_appearance

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request

Converts blob overmind/shield to update_appearance

## Why It's Good For The Game

Apparently the air_update_turf code earned it some ire. This should be better.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/f5bfb6c5-7b84-4137-920f-c483ab800f50

</details>

## Changelog
:cl: rkz
code: converted blob shield code to update_appearance
/:cl:

### CODE REVIEW

**Review by itsmeow — APPROVED** [review]
yipeeee

File: code/modules/antagonists/blob/structures/shield.dm [hunk 1]
```diff
@@ -19,19 +19,23 @@
...
+
+/obj/structure/blob/shield/update_desc(updates)
+	. = ..()
+	desc = (obj_integrity < (max_integrity * 0.5)) ? "[damaged_desc]" : initial(desc)
+
+/obj/structure/blob/shield/take_damage(damage_amount, damage_type, damage_flag, sound_effect, attack_dir)
+	. = ..()
+	if(. && obj_integrity > 0)
+		atmosblock = obj_integrity < (max_integrity * 0.5)
+		air_update_turf(1, atmosblock)
```
Comment (HowToLoLu) [inline code/modules/antagonists/blob/structures/shield.dm]: `air_update_turf` doesn't take two arguments it only takes 1, no need to pass it `atmosblock`

**Review by itsmeow — CHANGES_REQUESTED** [review]
fix `air_update_turf` (and use TRUE instead of 1)

File: code/modules/antagonists/blob/structures/shield.dm (hunk 1 above)
Comment (Tsar-Salat) [inline code/modules/antagonists/blob/structures/shield.dm]:
```suggestion
		air_update_turf(TRUE)
```

**Review by itsmeow — APPROVED** [review]

**Review by Penwin0 — APPROVED** [review]

**Pull request merged**